src/qmapshack/poi/CPoiDraw.cpp

```diff
@@ -137,6 +138,36 @@ void CPoiDraw::loadPoiPath(QSettings &cfg)
     poiPaths = cfg.value("poiPaths", poiPaths).toStringList();
 }
 
+poi_t CPoiDraw::findPOICloseBy(const QPoint &px) const
+{
+    poi_t poi;
+    CPoiItem::mutexActivePois.lock();
+    if(poiList)
+    {
+        for(int i = 0; i < poiList->count(); i++)
+        {
+            auto item = poiList->item(i);
```

Comment:
I do not really like auto. Sometimes it's really nice however IDEs can't resolve `auto`. This makes the code hard to read. Please stick with explicit types as long as possible.